.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+<!-- Remove the sections that are not applicable -->
+
+## Describe your changes
+
+<!-- Descriptions can consist of describing the change and/or images of the change -->
+<!-- Refrain from posting anything that refers to internal information -->
+
+| Before               | After               |
+| -------------------- | ------------------- |
+| Paste original image | Paste changed image |
+
+## Checklist before requesting a review
+
+- [ ] Is the PR title and/or first commit semantic? <!-- refer: https://github.com/box/box-ui-elements/blob/master/CONTRIBUTING.md#commit-types -->
+- [ ] Have you checked if the tests pass?
+- [ ] Did the build on CircleCi pass?
+- [ ] Are the checks on GitHub passing?
+
+## Checklist making a change
+
+- [ ] Did you verify the changes against the mocks?
+- [ ] Are your changes a11y compliant?
+- [ ] Are your changes responsive? (if applicable)
+- [ ] Did you test your changes in a parent app?
+
+## Checklist if you are adding new stories
+
+- [ ] Have you run `yarn start:storybook` to manually test your changes?
+- [ ] Did you ensure you only added necessary changes for adding a story?
+
+## Checklist if you are converting JS files to TS
```

Comment:
Write new storybook/TS readmes ?

Reply:
potentially link them instead of including them here directly